### DISCUSSION TITLE
airgap install of 2.3.80 : createrepo fails

### DISCUSSION THREAD

**Discussion by jpl33** [Security-Onion-Solutions/securityonion#6375] [Unsupported Versions] [answered]
call to createrepo in so-setup fails when I try airgap iso install of ver. 2.3.80.
the machine is running  ver. 2.3.21 currently. I am running the iso from a running instance.
should I try running it from boot?

thanks
Johnatan

**Comment by dougburks** [comment, marked as answer]
> call to createrepo in so-setup fails when I try airgap iso install of ver. 2.3.80.
> the machine is running ver. 2.3.21 currently. I am running the iso from a running instance.
> should I try running it from boot?

I'm not sure what you mean here.  

You can either:

- try running [soup and specifying the path to the ISO image to do an in-place upgrade](https://docs.securityonion.net/en/2.3/soup.html#airgap) 

OR 

- you can [boot from the ISO image to wipe and reinstall from scratch](https://docs.securityonion.net/en/2.3/installation.html#installation-using-security-onion-iso-image).

Also note that 2.3.90 is now available:
https://blog.securityonion.net/2021/11/security-onion-2390-wazuh-hotfix-now.html
